migrations/m220217_085029_entry_options.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2022 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+use humhub\components\Migration;
+
+/**
+ * Class m220217_085029_entry_options
+ */
+class m220217_085029_entry_options extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $type = $this->tinyInteger(1)->defaultValue(0)->notNull()->unsigned();
+        $this->safeAddColumn('calendar_entry', 'recurring', $type->after('all_day'));
```

Comment:
@luke- Done in commit https://github.com/humhub/calendar/pull/264/commits/669a52f3565c884e22127c1d86e7c7d9806d060b

But please note after this change it is impossible to disable all these setting by single uncheck action of the options, i.e. the checkboxes will be always enabled when data exists in DB. Or maybe we should clear the data from DB if the checkboxes were unchecking during update?

Reply:
@luke- Ok, I have decided to clear all recurring and reminder settings if their checkboxes are unchecked on the edit form - https://github.com/humhub/calendar/pull/264/commits/dddc0b758095ac5df731debf31298adc54a2e11c.

Please note if a checkbox(reminder or checked) is checked but the settings are not filled, then on next opening of the edit form the checkbox will be unchecked because currently we don't store the checkbox values in DB, they are checked only when their setting are filled with not null value.